From 46eddb497d9f0ccf5cf6d5d7aee9aa66511ce477 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 08 May 2023 18:13:17 +0300
Subject: [PATCH] [SW-0] fw_upd: staticize the accessor functions

The command reading and status writing functions are used only inside
the fw_upd module, so staticize them.

Change-Id: Id1dfe3ff13fb3d515ca60207251c273b473c1222
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/lib/preboot/module_fwupd.c b/lib/preboot/module_fwupd.c
index 702770e..013cdf6 100644
--- a/lib/preboot/module_fwupd.c
+++ b/lib/preboot/module_fwupd.c
@@ -63,22 +63,22 @@
 static struct fwupd_state i2c_state;
 static struct fwupd_resp_prop resp_props[RESP_PROP_NUM];
 
-u32 fwupd_cmd_readl(void)
+static u32 fwupd_cmd_readl(void)
 {
 	return regmap_readl(global_conf.upd_cmd_map);
 }
 
-void fwupd_sts_writel(u32 val)
+static void fwupd_sts_writel(u32 val)
 {
 	regmap_writel(val, global_conf.upd_sts_map);
 }
 
-u32 i2c_fwupd_cmd_readl(void)
+static u32 i2c_fwupd_cmd_readl(void)
 {
 	return i2c_state.cmd_reg;
 }
 
-void i2c_fwupd_sts_writel(u32 val)
+static void i2c_fwupd_sts_writel(u32 val)
 {
 	i2c_state.sts_reg = val;
 }
